pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: shelf
-version: 0.7.5
+version: 0.7.5+1
```

Comment:
Unneeded, since this CL just updates tests!

Reply:
Done!